lib/src/op_walk.rs

```diff
@@ -14,6 +14,8 @@
 
 //! Utility for operation id resolution and traversal.
 
+#![allow(missing_docs)]
```

Comment:
nit: can you move this to `OpsetEvaluationResult` type? or add short doc comment suppress warning?

Reply:
```rust
#[allow(missing_docs)]
pub type OpsetEvaluationResult<T> = Result<T, OpsetEvaluationError>;
```
or
```rust
/// doc comment here ...
pub type OpsetEvaluationResult<T> = Result<T, OpsetEvaluationError>;
```